frontend/src/components/UI/Modal/Modal.jsx

```diff
@@ -70,7 +71,14 @@ export const Modal = ({
                                 >
                                     {heading}
                                 </h2>
-                                {description && (
+                                {description && description instanceof Array && (
```

Comment:
wonder if we need to check array length is more than one here?
`description.length > 0 && (
// render the thing
)`